pkg/validations/extendedversion.go

```diff
@@ -31,3 +51,60 @@ func validateBundleSignature(bundle *v1alpha1.Bundles) error {
 	}
 	return nil
 }
+
+func isExtendedSupport(clusterSpec *anywherev1.Cluster, bundle *v1alpha1.Bundles) (bool, error) {
```

Comment:
this function only uses `clusterSpec.Spec.KubernetesVersion` from `clusterSpec` I recommend not taking in the whole `anywherev1.Cluster` object just for the kubernetes version field. I recommend just taking in what this function needs, so just the data type of `clusterSpec.Spec.KubernetesVersion`.